ci-operator/config/opendatahub-io/opendatahub-operator/opendatahub-io-opendatahub-operator-feature-rearchitecture.yaml

```diff
@@ -54,6 +54,27 @@ tests:
       IMAGE_REPO: opendatahub-operator
       RELEASE_VERSION: feature-rearchitecture
     workflow: opendatahub-io-ci-image-mirror
+- as: opendatahub-operator-e2e
+  steps:
+    cluster_profile: gcp
+    env:
+      COMPUTE_NODE_TYPE: n2-standard-8
+      OO_CHANNEL: fast
+      OO_INSTALL_NAMESPACE: openshift-operators
+      OO_PACKAGE: opendatahub-operator
+      OO_TARGET_NAMESPACES: '!all'
+    test:
+    - as: e2e
+      cli: latest
+      commands: |
+        unset GOFLAGS
+        make e2e-test -e OPERATOR_NAMESPACE=openshift-operators
```

Comment:
get me wondered why in makefile, we set `openshift-operators-system` as fallback value
if we change to use `openshift-operators` then we do not need to pass variable here.

Reply:
Its `opendatahub-operator-system` which is also created by default by sdk. I think we do not want local dev deployments to deploy in `openshift-operators` namespace always.